moodle-scalable-cluster-ubuntu/azuredeploy.json

```diff
@@ -587,6 +678,7 @@
             "dbServerType": "[parameters('dbServerType')]",
             "dbUsername": "[concat(parameters('dbLogin'), '@', parameters('dbServerType'), '-', variables('resourceprefix'))]",
             "elasticVmSku": "[parameters('elasticVmSku')]",
+            "dbDNS": "[if(equals(parameters('dbServerType'),'mssql'),concat(parameters('dbServerType'), '-',variables('resourcePrefix'), '.database.windows.net'), concat(parameters('dbServerType'), '-', variables('resourcePrefix'), '.', parameters('dbServerType'), '.database.azure.com'))]",
```

Comment:
use reference() to get the FQDNs

Reply:
According to the [documentation](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-template-functions-resource#reference), we can't use `reference()` in the variables section (mentioned in Remarks). Is this changed?

Reply:
you can't you need to move this into the properties section that needs it.

Reply:
Fixed in the newly added commit